spec/lib/common/json-schema-validator-spec.js

```diff
@@ -53,44 +53,173 @@ describe('JsonSchemaValidator', function () {
 
     helper.after();
 
-    it('should return true when validation pass', function () {
-        var result = validator.validate(testSchema1, { repo : '/172.31.128.1/mirrors' });
-        expect(result).to.be.true;
+    describe('validate', function() {
```

Comment:
I have separated the `describe` section for each method. So there will be conflict to resolve here. (if my PR goes in first)